src/swsssdk/scripts/sonic-db-cli

```diff
@@ -80,6 +81,8 @@ def execute_cmd(dbname, cmd, namespace, use_unix_socket=False):
             print()
         elif isinstance(resp, list):
             print("\n".join(resp))
+        elif isinstance(resp, dict):
+            print(json.dumps(resp))
```

Comment:
There is a design principle at the birth of this script
> sonic-db-cli output format mimic the non-tty mode output format from redis-cli

In order to easily replace the usage of `redis-cli` with `sonic-db-cli`. The only exception is the `list` type (such as output  of `keys`)
1. `redis-cli` outputs with blanks as delimitator
2. `sonic-db-cli` outputs with '\n' as delimitator
3. Fortunately all the parsing code of the output is not impacted

I see currently the dict type is not well implemented according to the design principle.
```
$ docker exec database redis-cli -n 4 hgetall "PORT|Ethernet20"
admin_status
up
alias
etp6
description
NEIGH06T2:Ethernet1
index
6
lanes
20,21,22,23
mtu
9100
pfc_asym
off
speed
40000
```

You proposed a new feature instead of a bug fix. I suggest add it with new command line options like:
```
$ sonic-db-cli --json CONFIG_DB HGETALL "TELEMETRY|gnmi"
```

Reply:
I agree. But I'd like to drop this PR since we could use ast.literal_eval to de-serialize dict from string.